Adding important note to solve #3493 (#3757)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md

-   Key trust

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

## Directory Synchronization

In hybrid deployments, users register the public portion of their Windows Hello for Business credential with Azure.  Azure AD Connect synchronizes the Windows Hello for Business public key to Active Directory.  

### Group Memberships for the Azure AD Connect Service Account
>[!IMPORTANT]

> > If you already have a Windows Server 2016 domain controller in your domain, you can skip Configure Permissions for Key Synchronization. For more please check [Configure Hybrid Windows Hello for Business: Directory Synchronization](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync)

Reviewer 2: ```suggestion
> If you already have a Windows Server 2016 domain controller in your domain, you can skip **Configure Permissions for Key Synchronization**. For more detail see [Configure Hybrid Windows Hello for Business: Directory Synchronization](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync).
```
